backend/frontend/templates/frontend/user/board/partial_card.html

```diff
@@ -1,170 +1,225 @@
                 </div>
-            {% endif %}
-            {% if card.code_review_ny_competent_since_last_review_request %}
-                <div class="flex" title="Number of competent code reviews since last review request">
-                    <div>
-                        <i class="fa-regular fa-face-frown fa-lg"></i>
-                    </div>
-                    <div class="font-bold bg-red-600 text-white rounded-full flex justify-center items-center h-5 px-2 text-xs -translate-x-1/2 -translate-y-2/4 top-0 right-0 mb-0.8 ml-1">{{card.code_review_ny_competent_since_last_review_request}} 
-                    </div>
+             {% endif %}
+             {% if card.code_review_ny_competent_since_last_review_request %}
+                <div class="relative inline-flex items-center p-2 text-sm font-medium text-center bg-opacity-10 rounded-lg">
+                  <i class="fa fa-frown-o" aria-hidden="true" title="Not yet competent reviews"></i>
+                  <div class="absolute inline-flex items-center justify-center w-5 h-5 text-xs text-white bg-red-600 border-2 border-white rounded-full -top-2 -end-2">{{ card.code_review_ny_competent_since_last_review_request }}</div>
                 </div>
-            {% endif %}
-            {% if card.code_review_red_flag_since_last_review_request %}
-                <div class="flex" title="Number of competent code reviews since last review request">
-                    <div>
-                        <i class="fa-regular fa-face-sad-tear fa-lg"></i>
-                    </div>
-                    <div class="font-bold bg-red-600 text-white rounded-full flex justify-center items-center h-5 px-2 text-xs -translate-x-1/2 -translate-y-2/4 top-0 right-0 mb-0.8 ml-1">{{card.code_review_red_flag_since_last_review_request}} 
-                    </div>
-                </div>    
-            {% endif %}
-        </div>
-    {% endif %}
+             {% endif %}
+             {% if card.code_review_red_flag_since_last_review_request %}
+                <div class="relative inline-flex items-center p-2 text-sm font-medium text-center bg-opacity-10 rounded-lg">
+                  <i class="fa-regular fa-face-sad-tear" aria-hidden="true" title="Red-flag reviews"></i>
+                  <div class="absolute inline-flex items-center justify-center w-5 h-5 text-xs text-white bg-red-600 border-2 border-white rounded-full -top-2 -end-2">{{ card.code_review_red_flag_since_last_review_request }}</div>
+                </div>   
+             {% endif %}
+             {% if card.code_review_excellent_since_last_review_request %}
+                <div class="relative inline-flex items-center p-2 text-sm font-medium text-center bg-opacity-10 rounded-lg">
+                  <i class="fa-regular fa-face-laugh-wink" aria-hidden="true" title="Excellent reviews"></i>
```

Comment:
This is looking very repetitive. Can you try make it more DRY?

This could help: https://github.com/carltongibson/django-template-partials

Reply:
If it seems like a difficult thing to do or it will cause too many delays, please add a new Mom task for it. You can label it as technical debt. We really should try hard not to write repetitive code.

Reply:
Technical debt issue added: https://umuzi-projects.monday.com/boards/1274815998/pulses/1398535402